src/components/UserManagement/UserTableFooter.jsx

```diff
@@ -91,8 +91,11 @@ const PageSummaryLabel = React.memo(props => {
       className="table-summary col-md-4 col-sm-4 col-xs-4 ember-view"
       style={{ marginBottom: '25px' }}
     >
-      {`Show ${(props.selectedPage - 1) * props.pageSize + 1} -
-    ${props.selectedPage * props.pageSize} of ${props.datacount}`}
+      {`Show `}
+      {props.datacount > props.selectedPage * props.pageSize
```

Comment:
Maybe it's more readability write like this:
``` Javascript
const displayedUsers = props.datacount > props.selectedPage * props.pageSize
    ? props.selectedPage * props.pageSize
    : props.datacount;
```

Reply:
thank you